drivers/sensor/nrf5/Kconfig

```diff
@@ -8,13 +8,3 @@ menuconfig TEMP_NRF5
 	depends on SOC_FAMILY_NRF
 	help
 	  Enable driver for nRF5 temperature sensor.
-
-if TEMP_NRF5
```

Comment:
After the only symbol dependent on the `TEMP_NRF5` option is removed, the option itself should be changed from `menuconfig` to `config`. See #20005.